jsonpath_ng/jsonpath.py

```diff
@@ -615,7 +618,14 @@ def filter(self, fn, data):
         return data
 
     def __str__(self):
-        return ','.join(map(str, self.fields))
+        # If any JsonPathLexer.literals are included in field name need quotes
+        # This avoids unnecessary quotes to keep strings short.
+        literals = JsonPathLexer.literals
```

Comment:
I think it's cleaner to just use `JsonPathLexer.literals` below rather than create a new variable.

Reply:
Done.